src/extension/devtools/devtools.ts

```diff
@@ -100,20 +90,11 @@ function startRequestInterval(ms = 500) {
     }
   }
 
-  if (devtoolsMachine.state.value === "connected") {
```

Comment:
We already check this conditional in other places that call this function, so we don't need to check it again here.